crates/env/src/api.rs

```diff
@@ -903,3 +903,56 @@ where
         instance.unlock_delegate_dependency::<E>(code_hash)
     })
 }
+
+/// Execute an XCM message locally, using the contract's address as the origin.
+///
+/// For more details consult
+/// [host function documentation](https://paritytech.github.io/substrate/master/pallet_contracts/api_doc/trait.Current.html#tymethod.xcm_execute).
+///
+/// # Errors
+///
+/// - If the message cannot be properly decoded on the pallet contracts side.
+/// - If the runtime doesn't allow for the contract unstable feature.
```

Comment:
What do you mean by this?

Reply:
The xcm host fns are still marked as unstable for now.
It's merged in master but I am not sure it has been audited. @athei  does this need to happen before we remove the unstable flag?

Reply:
In the past we marked them stable once end-to-end tests are there. Since you have those in this PR, I'd say we can mark it as stable once merged.